src/Tribe/Linked_Posts.php

```diff
@@ -182,8 +182,7 @@ public function get_meta_key( $post_type ) {
 	 * @return bool|string
 	 */
 	public function get_order_meta_key( $post_type ) {
-
-		if ( 'tribe_organizer' === $post_type ) {
+		if ( Tribe__Events__Main::ORGANIZER_POST_TYPE === $post_type ) {
```

Comment:
Lets use `Tribe__Events__Organizer::POSTTYPE` constant, we are trying to deprecated this one.